task9/delivery/meteorController.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express'
+import { getMeteorDto } from '../use_cases/meteorMapper.js'
+import { config } from "../config/config.js";
+
+const meteorRouter = express.Router()
+
+meteorRouter.get('/meteors', async (request, response) => {
+    try {
+        const { date, count, wereDangerousMeteors } = request.query;
+        const { startDate, endDate } = getDateRange(date)
+        const meteorDto = await getMeteorDto(startDate, endDate, wereDangerousMeteors, count);
```

Comment:
wereDangerousMeteors and count boolean data types. Convert please strings to boolean type

Reply:
done